### PR TITLE
Source Prestashop: Fix schema, remove empty datetime fields

### PR DESCRIPTION
**Pull request by artem1205**
## What

Fix Source Prestashop

## How
Fix expected records

## Recommended reading order
1. `components.py`
2. `manifest.yaml`

## 🚨 User Impact 🚨

quantity schema changed; schema update required

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/combinations.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
       "type": "string"
     },
     "quantity": {
-      "type": "string"
+      "type": ["null", "integer"]
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/combinations.json:27]: this is a breaking change :(

**Review by davydov-d — APPROVED** [review]
we should bump major version here

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`7a8b462a25`](https://github.com/airbytehq/airbyte/pull/27716/commits/7a8b462a25255aab91c2294b6c2cd815ba6876d2)) - ❌

⏲️  Total pipeline duration: 1273 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687805488/7a8b462a25255aab91c2294b6c2cd815ba6876d2/source-prestashop/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`7a8b462a25`](https://github.com/airbytehq/airbyte/pull/27716/commits/7a8b462a25255aab91c2294b6c2cd815ba6876d2)) - ❌

⏲️  Total pipeline duration: 1201 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687806899/7a8b462a25255aab91c2294b6c2cd815ba6876d2/source-prestashop/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`de462156a6`](https://github.com/airbytehq/airbyte/pull/27716/commits/de462156a6a0772ebf742bdfe393a20f3c109232)) - ❌

⏲️  Total pipeline duration: 1121 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687808180/de462156a6a0772ebf742bdfe393a20f3c109232/source-prestashop/0.4.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`16bf98dd4c`](https://github.com/airbytehq/airbyte/pull/27716/commits/16bf98dd4cbd700672b9a3ebfc8919a215c5d9f4)) - ❌

⏲️  Total pipeline duration: 1087 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687809502/16bf98dd4cbd700672b9a3ebfc8919a215c5d9f4/source-prestashop/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`ca63723e0c`](https://github.com/airbytehq/airbyte/pull/27716/commits/ca63723e0c4716054e76c24e86b0445a7242a03d)) - ❌

⏲️  Total pipeline duration: 83 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687868475/ca63723e0c4716054e76c24e86b0445a7242a03d/source-prestashop/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`ca63723e0c`](https://github.com/airbytehq/airbyte/pull/27716/commits/ca63723e0c4716054e76c24e86b0445a7242a03d)) - ❌

⏲️  Total pipeline duration: 62 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687868691/ca63723e0c4716054e76c24e86b0445a7242a03d/source-prestashop/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`831d25c0d3`](https://github.com/airbytehq/airbyte/pull/27716/commits/831d25c0d33b5554014c8e7ea4112aa8b689cfdf)) - ❌

⏲️  Total pipeline duration: 1090 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687870558/831d25c0d33b5554014c8e7ea4112aa8b689cfdf/source-prestashop/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Comment by octavia-squidington-iii** [comment]
##  source-prestashop test report (commit [`9cd3e19545`](https://github.com/airbytehq/airbyte/pull/27716/commits/9cd3e19545b9a459455ae779eab21f5b0dd2c845)) - ❌

⏲️  Total pipeline duration: 1104 seconds

| Step                                                                     | Result   |
|:-------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-prestashop/metadata.yaml | ✅        |
| Connector version semver check                                           | ✅        |
| Connector version increment check                                        | ✅        |
| QA checks                                                                | ✅        |
| Code format checks                                                       | ✅        |
| Connector package install                                                | ✅        |
| Build source-prestashop docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                               | ✅        |
| Integration tests                                                        | ✅        |
| Acceptance tests                                                         | ❌        |

🔗 [View the logs here](https://storage.googleapis.com/airbyte-ci-reports/airbyte-ci/connectors/test/pull_request/artem1205_con-health-prestashop/1687872094/9cd3e19545b9a459455ae779eab21f5b0dd2c845/source-prestashop/1.0.0/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-prestashop test
```

**Pull request merged**